_data/projects/OpinionatedGuides.yml

```diff
@@ -0,0 +1,11 @@
+name: Opinionated Guides
+desc: a one-stop guide to computers, linux, and electronics.
+site: https://github.com/opinionatedguide/opinionatedguide.github.io-src
```

Comment:
This link doesn't need to be the code, and could be to the site itself, e.g. `https://opinionatedguide.github.io/`